modules/core/src/main/scala/queryinterpreter.scala

```diff
@@ -170,7 +170,7 @@ class QueryInterpreter[F[_]](mapping: Mapping[F]) {
             case Ior.Right(elem) => (errors, elem :: elems)
             case Ior.Both(errs, elem) => (errs.toChain ++ errors, elem :: elems)
           }
-      }).map(_.reverse)
+      }).fmap(_.reverse)
```

Comment:
Functor `.map` syntax doesn't work for tuples in Scala 3 because there's polymorphic `map` on `Tuple` now.